src/Cake.Core/Extensions/CakeArgumentsExtensions.cs

```diff
@@ -17,10 +17,11 @@ public static class CakeArgumentsExtensions
         /// </remarks>
         /// <param name="arguments">The arguments.</param>
         /// <param name="name">The argument name.</param>
+        /// <param name="defaultValue">The default value to use if the argument is not found.</param>
         /// <returns>The argument value.</returns>
-        public static string GetArgument(this ICakeArguments arguments, string name)
+        public static string GetArgument(this ICakeArguments arguments, string name, string defaultValue = null)
         {
-            return arguments.GetArguments(name).LastOrDefault();
+            return arguments.HasArgument(name) ? arguments.GetArguments(name).LastOrDefault() : defaultValue;
```

Comment:
Adding an optional parameter will break library ABI, to not break ABI an overload should be added instead, something like 
```csharp
 public static string GetArgument(this ICakeArguments arguments, string name, string defaultValue)
   => arguments.GetArgument(name) ?? defaultValue;
```

Reply:
I see, done!